### ISSUE TITLE
Autocorrection bug in `Style/RedundantConditional` cop

### ISSUE THREAD

**Issue by pocke** [bbatsov/rubocop#4676]
If "bad" code has `elsif`, `Style/RedundantConditional` generates wrong code when autocorrect.

```ruby
if cond
  foo
elsif x == y
  true
else
  false
end
```

--------

## Expected behavior

Autocorrection generates the following code.

```ruby
if cond
  foo
else
  x == y
end
```

## Actual behavior

Autocorrection generates the following code actually.

```ruby
if cond
  foo
  x == y
end
```

## Steps to reproduce the problem

1. Put the above code as `test.rb`
2. Run `rubocop -a`

## RuboCop version

master(ce6483c8f83691837519ab0a4f996a74f9de11dc)

**Comment by Drenmi** [comment]
Good catch! I can see this happening with how `parser` nests the AST for `elsif`s.
